CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Fixed`
 
 - [#428](https://github.com/nf-core/ampliseq/pull/428) - Fixed samplesheet sampleID entries, now allows dashes.
+- [#433](https://github.com/nf-core/ampliseq/pull/433) - Fixed typo's and improved documentation layout.
```

Comment:
There's a typo here: "typo's" should be "typos" as it's plural, not genitive! :-)

Reply:
True, thanks!